Net22Solution/Net22Task/XmlLoginValidator.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+using System.Xml;
+using System.Xml.Linq;
+
+namespace Net22Task
+{
+    public  class XmlLoginValidator
```

Comment:
Model classes (Config / Window) have validation methods. 
This class duplicates functionality and adds validation methods as well. 

Keep validation only at models classes or only in XmlLoginValidator.